### PR TITLE
docs(readme): Add deprecation notice

### LINKED ISSUE #22: Deprecate this repo
Deprecate this repo in favor of this task https://github.com/commercetools/nodejs/issues/109

### PR DESCRIPTION
**Pull request by junajan**
This repo can be deprecated because the code was moved here https://github.com/commercetools/nodejs/issues/117

Resolves #22

### CODE REVIEW

**Review by wizzy25 — APPROVED** [review]
Good job 👍 
I think this should be on hold though, until commercetools/nodejs#117 gets merged and released

**Review by adnasa — APPROVED** [review]

**Review by hisabimbola — APPROVED** [review]

**Pull request merged**
